.circleci/config.yml

```diff
@@ -1,166 +1,88 @@
-
-      - run: go version
+      - *restore-godel-cache
       - run: ./godelw version
-      - run: ./godelw verify --apply=false --junit-output="${ARTIFACT_STORE}/tests.xml"
-
-      - setup_remote_docker
-      - *install-docker-client
-
-      - run: ./godelw dist
-      - *make-client
-      - *build-docker-image
-
-      - save_cache:
-          paths:
-            - ./client/node_modules
-          key: v1-dependencies-{{ checksum "./client/package.json" }}
+      - *save-godel-cache
+      - run: mkdir -p "${TESTS_DIR}"
+      - run: ./godelw build
+      - run: ./godelw verify --apply=false --junit-output="$TESTS_DIR/$CIRCLE_PROJECT_REPONAME-tests.xml"
+      - *store-results
+      - *store-artifacts
       - persist_to_workspace:
-          root: /go/src/github.com/palantir/bulldozer
+          root: .
           paths:
-            - client/build
-      - *store-test-results
-
-  changelog:
-    <<: *darwin-linux-no-cgo
-    steps:
-      - checkout
-      - *install-git
-      - *define-artifacts-dir
-      - *mkdir-artifacts-dir
-      - run: git log `git describe --tags --abbrev=0`..HEAD --pretty="### %s%n%b%n" > /tmp/artifacts/changelog.md
-      - *store-artifacts
+            - build/bulldozer
 
-  publish-snapshot:
-    <<: *darwin-linux-no-cgo
+  dist:
+    working_directory: /go/src/github.com/palantir/bulldozer
+    docker:
+      - image: nmiyake/go:go-darwin-linux-1.11-docker-17.06.0-ce-bsdtar-unzip-t134
     steps:
       - checkout
       - setup_remote_docker
-      - *install-docker-client
       - attach_workspace:
-          at: /go/src/github.com/palantir/bulldozer
-      - run: ./godelw dist
-      - *build-docker-image
-      - run: ./scripts/publish_docker.sh
-
-  publish-release:
-    <<: *darwin-linux-no-cgo
-    steps:
-      - checkout
-      - setup_remote_docker
-      - *install-docker-client
-      - run: ./godelw publish bintray --url https://api.bintray.com --subject palantir --repository releases --user "$BINTRAY_USERNAME" --password "$BINTRAY_PASSWORD" --publish --downloads-list bulldozer
-      - run: sha256sum dist/*.tgz
-      - attach_workspace:
-          at: /go/src/github.com/palantir/bulldozer
-      - *build-docker-image
-      - run: ./scripts/publish_docker.sh
+          at: .
+      - *restore-godel-cache
+      - run: ./godelw docker build --verbose
+      - deploy:
+          name: Publish
+          command: |
+            if [[ "${CIRCLE_BRANCH}" == "develop" ]]; then
+              docker login -u "${DOCKERHUB_USERNAME}" -p "${DOCKERHUB_PASSWORD}"
+              ./godelw docker push --tags=snapshot
+            fi
+            if [[ -n "${CIRCLE_TAG}" ]] ; then
+              docker login -u "${DOCKERHUB_USERNAME}" -p "${DOCKERHUB_PASSWORD}"
+              ./godelw docker push --tags=latest,version
+              ./godelw publish bintray \
+                  --url https://api.bintray.com \
+                  --subject palantir --repository releases --product policy-bot \
```

Comment:
Need to fix the `policy-bot` references in this command (`--product` and `--publish`).